src/components/Filters/components/FilterListContainer/FilterListContainer.scss

```diff
@@ -0,0 +1,86 @@
+@use '@gravity-ui/uikit/styles/old-variables/spacing';
+@use '@gravity-ui/uikit/styles/mixins';
+
+@use 'src/components/variables';
+
+$block: '.#{variables.$ns}filter-list-container';
+
+#{$block} {
+    &__search {
+        padding: var(--g-spacing-1) var(--g-spacing-1) 0;
+    }
+
+    &__list {
+        max-height: 45vh;
+        overflow: auto;
+
+        &_virtualized {
+            --yc-list-height: 256px;
+            max-height: none;
+        }
+
+        &_mobile {
+            #{$block}__search {
+                flex: none;
+            }
+            .yc-list__items {
+                overflow-y: auto;
+                flex: auto;
+                min-height: 0;
+            }
+        }
+    }
+
+    &__item {
+        display: flex;
+        flex: auto;
+        justify-content: space-between;
+        align-items: center;
+
+        padding: 0 var(--g-spacing-4);
+
+        width: 100%;
+        margin-inline-end: var(--g-spacing-5);
+
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+
+        &_mobile {
+            @include mixins.text-body-2();
+            background: transparent;
+        }
+
+        &-description {
```

Comment:
U should named it like `&__item-description` and do not defined as a child of `&__item`